contentcuration/contentcuration/static/js/edit_channel/admin/views.js

```diff
@@ -259,7 +259,7 @@ var ChannelTab = BaseAdminTab.extend({
             label: "# of Items",
             filter: function(item1, asc, item2){
                 var count1 = item1.get("count");
-                var count2 = item1.get("count");
+                var count2 = item2.get("count");
```

Comment:
I was wondering why sorting by "# of Items" on the admin page wasn't sorting correctly, but it makes sense now 😬

Reply:
:microscope: